specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/examples/CosmosDBPrivateLinkResourceListGet.json

```diff
@@ -0,0 +1,28 @@
+{
+  "parameters": {
+    "subscriptionId": "subid1",
+    "resourceGroupName": "rg1",
+    "accountName": "ddb1",
+    "api-version": "2015-04-08"
```

Comment:
So you're not releasing a preview version of this resource prior to GA? Just want to make sure you're aware of the implications of releasing it with a stable api version. See https://armwiki.azurewebsites.net/api_contracts/APIDeprecationPolicy.html for some more info on this.